src/iuse.h

```diff
@@ -35,7 +35,6 @@ class iuse
     int ecig                ( player*, item*, bool, const tripoint& );
     int antibiotic          ( player*, item*, bool, const tripoint& );
     int eyedrops            ( player*, item*, bool, const tripoint& );
-    int fungicide           ( player*, item*, bool, const tripoint& );
```

Comment:
I think like this:
* No arm breaking at the end
* Instead, it tries to give you random mostly-negative fungal mutations (weaker than full mycus) like covering a random slot in fungal stalks, causing extra encumbrance if covered but providing minor protection from physical and acid
* After giving the mutation, infection stops.
* If you get reinfected while having a fungal mutation, negative effects of infection are changed to just pure warnings (ie. you taste shrooms instead of you vomit your guts out)
* If it picks a mutation you already have (because you got re-infected), it gives random mutation from a new fungal tree, containing mostly weak mutations like chitin skin, infection resistance, paws, cold-blooded, weak stomach, strong stomach etc.
* Vomiting while having any fungal mutation causes you to cause fungalization of furniture instead of bile tiles (mostly just flavor change)
* Anti-fungal drugs used with at least one fungal mutation cause giant drop of health, some non-lethal hp (if taken near full hp) damage and cure one mutation. Purifier shouldn't cure fungal mutations, but could cure those from fungal tree